libraries/typings-generator/package.json

```diff
@@ -23,13 +23,13 @@
   "dependencies": {
     "@rushstack/node-core-library": "workspace:*",
     "@rushstack/terminal": "workspace:*",
-    "chokidar": "~3.4.0",
+    "chokidar": "~3.6.0",
     "fast-glob": "~3.3.1"
   },
   "devDependencies": {
     "@rushstack/heft": "workspace:*",
-    "local-node-rig": "workspace:*",
-    "@types/glob": "7.1.1"
+    "@types/glob": "7.1.1",
```

Comment:
Why do we even still have this dependency? `fast-glob` provides its own types; is this for `chokidar`?

Reply:
Nope. Removed.